ipywidgets/widgets/widget.py.orig

```diff
@@ -0,0 +1,626 @@
+"""Base Widget class.  Allows user to create widgets in the back-end that render
```

Comment:
Ah, who is using `git commit -am` instead of git gui and looking at what they are committing? :)